ingenialink/servo.py

```diff
@@ -36,14 +36,16 @@ def __init__(self, servo):
 
     def run(self):
         """Checks if the drive is alive by reading the status word register"""
+        current_state = {}
```

Comment:
You can use dictionary comprehension to simplify these lines.

Reply:
Actually, we can use the `Servo.status` property which is exactly this dict.